css/styles.css

```diff
@@ -605,3 +627,135 @@ textarea::placeholder {
   height: 40px;
   gap: 18px;
 }
+
+.close-icon-detail {
+  background: url(../Images/icons/close_detail.png) no-repeat center/cover;
+  width: 12px;
+  height: 12px;
+}
+
+.project-details-popup.show {
+  display: block;
+}
+
+/* ====================================== */
+.project-details-popup {
+  overflow: hidden;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  position: fixed;
+  height: 105vh;
+  width: 95.5vw;
+  top: 0;
+  padding-left: 2.25vw;
+  display: none;
+}
+
+.tech-list {
+  display: flex;
+  bottom: 1rem;
+  flex-direction: row;
+  gap: 1rem;
+  justify-content: flex-start;
+  list-style: none;
+}
+
+.popup-heading {
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.project-details-inner {
+  margin: auto;
+  background-color: #fff;
+  padding: 15px 16px;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 0 0 3px 0 black;
+}
+
+.pop-head {
+  align-self: stretch;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.project-details-header {
+  display: flex;
+  flex-direction: column;
+  gap: 33px 0;
+  align-self: stretch;
+}
+
+.modal-buttons {
+  display: flex;
+  justify-content: space-between;
+  gap: 0 20px;
+  margin-top: 150px;
+}
+
+.popup-content {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.popup-content img {
+  border: 2px inset #eee;
+}
+
+.popup-lower {
+  display: flex;
+  flex-direction: column;
+  align-self: self-start;
+  flex-wrap: wrap;
+}
+
+.live-icon {
+  background: url(../Images/icons/Icon-Export.png) no-repeat center center/cover;
+  width: 24px;
+  height: 24px;
+}
+
+.source-icon {
+  background: url(../Images/icons/Icon-GitHub.png) no-repeat center center/cover;
+  width: 24px;
+  height: 24px;
+}
+
+.modal-buttons div {
+  align-items: center;
+  gap: 15px;
+  display: flex;
+}
+
+.btn-details {
+  font-family: "Inter", sans-serif;
+  border: 1px solid transparent;
+  line-height: 24px;
+  font-size: 17px;
+  padding: 12px;
+  font-weight: 700;
+  background-color: #f55800;
+}
+
+.btn-details:hover {
+  border: 1px solid #f55800;
+  background-color: #fff5e1;
+  color: #f55800;
+}
+
+.btn-details:active {
+  background-color: #e05304;
+  color: #f4f1f0;
+}
+
+.btn-details:disabled {
+  border: 1px solid transparent;
+  color: #5b5957;
+  background-color: #f4f1f0;
+}
```

Comment:
As requested by the first reviewer, kindly `blur` the background of the `pop-up` so the stands out for a good user experience. See the screenshot below for more details:
Figma: 
<img width="1152" alt="Screenshot 2023-04-12 at 07 22 47" src="https://user-images.githubusercontent.com/107192076/231383636-bf1287a7-9340-4a0d-9427-356b0d48a99e.png">

Your design:
<img width="1363" alt="Screenshot 2023-04-12 at 07 22 22" src="https://user-images.githubusercontent.com/107192076/231385185-2f1f10c5-9ba6-4e67-901c-bedae75e01bb.png">

In the above screenshot kindly `blur` everything outside of the model where I have the text `please blur here`.

Reply:
Okay thank you @Strangeal